<a href="https://colab.research.google.com/github/HaJeong-K/Jeju_project/blob/main/2018_%EC%A0%84%EC%B2%98%EB%A6%AC.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd

from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


## 2018 년

# 데이터 전처리

데이터 수집 비율
- 2018년도 제주 여행자를 대상으로 한 방문 관광객 실태조사에서 응답자 프로파일을 가져옴.
 + 응답자 프로파일의 카테고리 중 월별 카테고리만 수집함.
- 해당 연도의 데이터 수집 대상 비율을 명시해 앞으로 다룰 데이터들이 어떤 방식으로 추출되었는지 설명함.

In [ ]:
import pandas as pd

from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


## 1인당 지출 경비 전처리
- 1인당 지출 경비에 대한 카테고리: 전체 부분, 숙박 부분, 식음료 부분, 항공/선박 부분, 관광/문화 지출 부분
- 1인당 지출 경비 부분이 지출 기준인원 항목의 하위 항목으로 분류되어 있어 응답자 프로파일처럼 해당 데이터의 비율을 명시함.

In [4]:
import pandas as pd

from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [5]:
expenditure_standard = pd.read_excel('/content/drive/MyDrive/2018 지출 기준 인원.xlsx')
expenditure_standard.head()

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0   NaN        NaN        사례수         1명         2명         3명         4명   
1  [전체]        NaN       6181       15.3       29.4       12.8       22.6   
2    월별        01월        492       33.2         24       11.3       20.7   
3    월별        02월        420       40.9       32.4        6.7         13   
4    월별        03월        486       24.6       32.4        9.1       17.3   

  Unnamed: 7 Unnamed: 8 Unnamed: 9 Unnamed: 10 Unnamed: 11 Unnamed: 12  \
0         5명         6명         7명          8명          9명         10명   
1        5.3        5.1        2.4         2.4         0.9           2   
2        4.6        2.3        0.3         0.5         0.3         0.8   
3        2.2        1.9        1.1         0.3           0         0.3   
4        5.3        3.5        1.1         1.9         1.6           1   

  Unnamed: 13 Unnamed: 14  
0      11명 이상        [평균]  
1         1.8        3.49  
2         1.8        2.77  
3         1.3        2.31  
4         2.1         3.1

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '1명',
           'Unnamed: 4' : '2명',
           'Unnamed: 5' : '3명',
           'Unnamed: 6' : '4명',
           'Unnamed: 7' : '5명',
           'Unnamed: 8' : '6명',
           'Unnamed: 9' : '7명',
           'Unnamed: 10' : '8명',
           'Unnamed: 11' : '9명',
           'Unnamed: 12' : '10명',
           'Unnamed: 13' : '11명 이상',
           'Unnamed: 14' : '평균'}

expenditure_standard = expenditure_standard.rename(columns = columns)
expenditure_standard

분석 카테고리       세부 카테고리   사례수    1명    2명    3명    4명    5명   6명   7명  \
0          NaN           NaN   사례수    1명    2명    3명    4명    5명   6명   7명   
1         [전체]           NaN  6181  15.3  29.4  12.8  22.6   5.3  5.1  2.4   
2           월별           01월   492  33.2    24  11.3  20.7   4.6  2.3  0.3   
3           월별           02월   420  40.9  32.4   6.7    13   2.2  1.9  1.1   
4           월별           03월   486  24.6  32.4   9.1  17.3   5.3  3.5  1.1   
5           월별           04월   576  18.8    31   9.1  26.6   4.5  4.6  1.1   
6           월별           05월   567  20.4  23.8  12.5  22.2   4.8  5.1  2.8   
7           월별           06월   554  10.4  28.6  10.2  20.1   6.9  8.6  6.8   
8           월별           07월   528   5.9  35.6  14.4  23.8   6.6  6.4  2.5   
9           월별           08월   540   7.6  34.7  14.6  21.6   6.8  5.5  3.2   
10          월별           09월   517   6.6  31.1  15.4    31   5.5  4.4  1.6   
11          월별           10월   531   8.3    20  16.4  25.9     6    5  2.2   
12          월별           11월   489   6.9    34  13.9  23.5   4.1  5.1  2.5   
13          월별           12월   479   5.8  26.3  19.3  23.2   4.8    8  3.2   
14         거주지         서울특별시  1467  16.2  30.6  11.7  23.8   6.2    4  2.5   
15         거주지         부산광역시   724    13  31.1   9.6  24.4   3.9  5.8  3.9   
16         거주지         대구광역시   538    12  27.9  15.2  22.9   6.7  6.5  3.4   
17         거주지         인천광역시   375  11.7    32  16.4  25.3   4.7  4.3  1.6   
18         거주지         광주광역시   215  19.6  25.3  14.8  21.2   6.6  3.4  3.1   
19         거주지         대전광역시   185  16.6  25.2    12  21.1   2.1  9.4  2.6   
20         거주지         울산광역시   187  16.6    27   9.4    24   5.2  5.1    4   
21         거주지         세종자치시    34  11.5  21.9   4.3  36.8    11  7.5    0   
22         거주지           경기도  1284  13.4  32.8  15.2  21.8   4.7  4.3  1.3   
23         거주지           강원도    99  25.7    20   9.3  22.3   2.7  6.1  1.2   
24         거주지           충청도   262  16.7    29   9.7  21.1   4.4  5.6  3.2   
25         거주지           전라도   385  18.9  22.7  12.5  21.2   5.8  5.1  1.9   
26         거주지           경상도   416    20  25.7  11.8  17.7   5.8  7.7  1.6   
27         거주지  해외거주 등 기타 지역    11  21.6  32.6  45.8     0     0    0    0   
28          성별            남자  2600  20.3  28.8   8.7  22.4   4.8    5  2.2   
29          성별            여자  3581  11.7  29.9  15.7  22.8   5.6  5.2  2.6   
30          연령     15세 - 20세   127  40.3  18.1   4.6  22.7   4.8  1.9  0.9   
31          연령     21세 - 30세  1412  17.4    54  11.3  11.1   2.9  1.7  0.4   
32          연령     31세 - 40세  1518  10.9  25.7  18.3  27.1   5.8    5  2.3   
33          연령     41세 - 50세  1250  12.4  17.4  14.3  32.5   7.3  5.8  3.4   
34          연령     51세 - 60세  1371  16.8    21  10.3  23.1   5.2  7.1  3.2   
35          연령        60세 이상   503    20  27.2     5  15.6   5.4  8.6  4.1   
36        방문목적     휴가 및 순수여행  5571  11.7  30.2  13.3  23.9   5.4  5.4  2.6   
37        방문목적        비즈니스여행   405  48.1    23   6.8  11.7   3.9  3.1  0.7   
38        방문목적          기타여행   204  48.5  20.4   9.1   8.8   4.7  1.8  1.2   
39        여행형태          개별여행  5884  12.4  30.5  13.4  23.5   5.4  5.2  2.5   
40        여행형태         패키지여행   248  84.5   4.5     1     3     1  2.1  0.5   
41        여행형태       부분패키지여행    49  25.1  22.5     0    17  10.3  2.5  2.6   
42        방문횟수            1회  1934  12.7  29.5    12  23.7   5.8  5.5  2.4   
43        방문횟수            2회  2483  10.9  29.9  13.7  25.2   5.3  4.9  2.8   
44        방문횟수            3회   813  17.3  29.8  13.5  20.1   5.6    5  2.3   
45        방문횟수         4회 이상   951  30.7  27.7  11.3  15.7   3.9  4.9  1.4   
46  ♠BASE : 전체           NaN   NaN   NaN   NaN   NaN   NaN   NaN  NaN  NaN   

     8명   9명  10명  11명 이상    평균  
0    8명   9명  10명  11명 이상  [평균]  
1   2.4  0.9    2     1.8  3.49  
2   0.5  0.3  0.8     1.8  2.77  
3   0.3    0  0.3     1.3  2.31  
4   1.9  1.6    1     2.1   3.1  
5   1.8  0.2  0.9     1.4  3.19  
6   2.1  2.1  1.6     2.

In [ ]:
expenditure_standard = expenditure_standard.drop(index = range(14, 30)).reset_index(drop=True)

expenditure_standard

분석 카테고리    세부 카테고리   사례수    1명    2명    3명    4명    5명   6명   7명   8명  \
0          NaN        NaN   사례수    1명    2명    3명    4명    5명   6명   7명   8명   
1         [전체]        NaN  6181  15.3  29.4  12.8  22.6   5.3  5.1  2.4  2.4   
2           월별        01월   492  33.2    24  11.3  20.7   4.6  2.3  0.3  0.5   
3           월별        02월   420  40.9  32.4   6.7    13   2.2  1.9  1.1  0.3   
4           월별        03월   486  24.6  32.4   9.1  17.3   5.3  3.5  1.1  1.9   
5           월별        04월   576  18.8    31   9.1  26.6   4.5  4.6  1.1  1.8   
6           월별        05월   567  20.4  23.8  12.5  22.2   4.8  5.1  2.8  2.1   
7           월별        06월   554  10.4  28.6  10.2  20.1   6.9  8.6  6.8  2.3   
8           월별        07월   528   5.9  35.6  14.4  23.8   6.6  6.4  2.5  1.9   
9           월별        08월   540   7.6  34.7  14.6  21.6   6.8  5.5  3.2    2   
10          월별        09월   517   6.6  31.1  15.4    31   5.5  4.4  1.6  2.3   
11          월별        10월   531   8.3    20  16.4  25.9     6    5  2.2  5.8   
12          월별        11월   489   6.9    34  13.9  23.5   4.1  5.1  2.5  4.1   
13          월별        12월   479   5.8  26.3  19.3  23.2   4.8    8  3.2  3.5   
14          연령  15세 - 20세   127  40.3  18.1   4.6  22.7   4.8  1.9  0.9  1.9   
15          연령  21세 - 30세  1412  17.4    54  11.3  11.1   2.9  1.7  0.4  0.5   
16          연령  31세 - 40세  1518  10.9  25.7  18.3  27.1   5.8    5  2.3  1.1   
17          연령  41세 - 50세  1250  12.4  17.4  14.3  32.5   7.3  5.8  3.4  2.6   
18          연령  51세 - 60세  1371  16.8    21  10.3  23.1   5.2  7.1  3.2    5   
19          연령     60세 이상   503    20  27.2     5  15.6   5.4  8.6  4.1  4.4   
20        방문목적  휴가 및 순수여행  5571  11.7  30.2  13.3  23.9   5.4  5.4  2.6  2.6   
21        방문목적     비즈니스여행   405  48.1    23   6.8  11.7   3.9  3.1  0.7  0.3   
22        방문목적       기타여행   204  48.5  20.4   9.1   8.8   4.7  1.8  1.2  1.2   
23        여행형태       개별여행  5884  12.4  30.5  13.4  23.5   5.4  5.2  2.5  2.5   
24        여행형태      패키지여행   248  84.5   4.5     1     3     1  2.1  0.5  0.1   
25        여행형태    부분패키지여행    49  25.1  22.5     0    17  10.3  2.5  2.6  7.4   
26        방문횟수         1회  1934  12.7  29.5    12  23.7   5.8  5.5  2.4  2.7   
27        방문횟수         2회  2483  10.9  29.9  13.7  25.2   5.3  4.9  2.8  2.5   
28        방문횟수         3회   813  17.3  29.8  13.5  20.1   5.6    5  2.3  2.5   
29        방문횟수      4회 이상   951  30.7  27.7  11.3  15.7   3.9  4.9  1.4  1.5   
30  ♠BASE : 전체        NaN   NaN   NaN   NaN   NaN   NaN   NaN  NaN  NaN  NaN   

     9명  10명  11명 이상    평균  
0    9명  10명  11명 이상  [평균]  
1   0.9    2     1.8  3.49  
2   0.3  0.8     1.8  2.77  
3     0  0.3     1.3  2.31  
4   1.6    1     2.1   3.1  
5   0.2  0.9     1.4  3.19  
6   2.1  1.6     2.6  3.59  
7   0.9  2.5     2.7     4  
8   0.1  1.5     1.4  3.51  
9   0.7  2.1     1.1  3.57  
10  1.2  0.7     0.2  3.43  
11    2  5.1     3.2  4.39  
12  0.8  3.8     1.5  3.75  
13  0.5  3.1     2.3  4.04  
14    1    1       3  2.93  
15  0.2  0.1     0.4  2.46  
16  0.6  1.9     1.2  3.49  
17    1  1.7     1.6  3.83  
18  1.5  3.4     3.4   4.1  
19  1.5  4.4     3.7  4.07  
20    1  2.1     1.8   3.6  
21    0  0.7     1.6  2.49  
22    0  1.8     2.5  2.57  
23  0.9    2     1.8  3.55  
24  0.1    0     3.2  1.98  
25  5.2  7.5       0  4.03  
26  1.1  3.1     1.5  3.65  
27    1  1.8     1.9  3.61  
28  0.5  0.8     2.7  3.42  
29  0.4    1     1.7  2.94  
30  NaN  NaN     NaN   NaN

In [ ]:
expenditure_standard = expenditure_standard.drop(index = range(20, 31)).reset_index(drop=True)

expenditure_standard

분석 카테고리    세부 카테고리   사례수    1명    2명    3명    4명   5명   6명   7명   8명   9명  \
0      NaN        NaN   사례수    1명    2명    3명    4명   5명   6명   7명   8명   9명   
1     [전체]        NaN  6181  15.3  29.4  12.8  22.6  5.3  5.1  2.4  2.4  0.9   
2       월별        01월   492  33.2    24  11.3  20.7  4.6  2.3  0.3  0.5  0.3   
3       월별        02월   420  40.9  32.4   6.7    13  2.2  1.9  1.1  0.3    0   
4       월별        03월   486  24.6  32.4   9.1  17.3  5.3  3.5  1.1  1.9  1.6   
5       월별        04월   576  18.8    31   9.1  26.6  4.5  4.6  1.1  1.8  0.2   
6       월별        05월   567  20.4  23.8  12.5  22.2  4.8  5.1  2.8  2.1  2.1   
7       월별        06월   554  10.4  28.6  10.2  20.1  6.9  8.6  6.8  2.3  0.9   
8       월별        07월   528   5.9  35.6  14.4  23.8  6.6  6.4  2.5  1.9  0.1   
9       월별        08월   540   7.6  34.7  14.6  21.6  6.8  5.5  3.2    2  0.7   
10      월별        09월   517   6.6  31.1  15.4    31  5.5  4.4  1.6  2.3  1.2   
11      월별        10월   531   8.3    20  16.4  25.9    6    5  2.2  5.8    2   
12      월별        11월   489   6.9    34  13.9  23.5  4.1  5.1  2.5  4.1  0.8   
13      월별        12월   479   5.8  26.3  19.3  23.2  4.8    8  3.2  3.5  0.5   
14      연령  15세 - 20세   127  40.3  18.1   4.6  22.7  4.8  1.9  0.9  1.9    1   
15      연령  21세 - 30세  1412  17.4    54  11.3  11.1  2.9  1.7  0.4  0.5  0.2   
16      연령  31세 - 40세  1518  10.9  25.7  18.3  27.1  5.8    5  2.3  1.1  0.6   
17      연령  41세 - 50세  1250  12.4  17.4  14.3  32.5  7.3  5.8  3.4  2.6    1   
18      연령  51세 - 60세  1371  16.8    21  10.3  23.1  5.2  7.1  3.2    5  1.5   
19      연령     60세 이상   503    20  27.2     5  15.6  5.4  8.6  4.1  4.4  1.5   

    10명  11명 이상    평균  
0   10명  11명 이상  [평균]  
1     2     1.8  3.49  
2   0.8     1.8  2.77  
3   0.3     1.3  2.31  
4     1     2.1   3.1  
5   0.9     1.4  3.19  
6   1.6     2.6  3.59  
7   2.5     2.7     4  
8   1.5     1.4  3.51  
9   2.1     1.1  3.57  
10  0.7     0.2  3.43  
11  5.1     3.2  4.39  
12  3.8     1.5  3.75  
13  3.1     2.3  4.04  
14    1       3  2.93  
15  0.1     0.4  2.46  
16  1.9     1.2  3.49  
17  1.7     1.6  3.83  
18  3.4     3.4   4.1  
19  4.4     3.7  4.07

In [ ]:
expenditure_standard = expenditure_standard.drop(index = range(14, 20)).reset_index(drop=True)

expenditure_standard

분석 카테고리 세부 카테고리   사례수    1명    2명    3명    4명   5명   6명   7명   8명   9명  \
0      NaN     NaN   사례수    1명    2명    3명    4명   5명   6명   7명   8명   9명   
1     [전체]     NaN  6181  15.3  29.4  12.8  22.6  5.3  5.1  2.4  2.4  0.9   
2       월별     01월   492  33.2    24  11.3  20.7  4.6  2.3  0.3  0.5  0.3   
3       월별     02월   420  40.9  32.4   6.7    13  2.2  1.9  1.1  0.3    0   
4       월별     03월   486  24.6  32.4   9.1  17.3  5.3  3.5  1.1  1.9  1.6   
5       월별     04월   576  18.8    31   9.1  26.6  4.5  4.6  1.1  1.8  0.2   
6       월별     05월   567  20.4  23.8  12.5  22.2  4.8  5.1  2.8  2.1  2.1   
7       월별     06월   554  10.4  28.6  10.2  20.1  6.9  8.6  6.8  2.3  0.9   
8       월별     07월   528   5.9  35.6  14.4  23.8  6.6  6.4  2.5  1.9  0.1   
9       월별     08월   540   7.6  34.7  14.6  21.6  6.8  5.5  3.2    2  0.7   
10      월별     09월   517   6.6  31.1  15.4    31  5.5  4.4  1.6  2.3  1.2   
11      월별     10월   531   8.3    20  16.4  25.9    6    5  2.2  5.8    2   
12      월별     11월   489   6.9    34  13.9  23.5  4.1  5.1  2.5  4.1  0.8   
13      월별     12월   479   5.8  26.3  19.3  23.2  4.8    8  3.2  3.5  0.5   

    10명  11명 이상    평균  
0   10명  11명 이상  [평균]  
1     2     1.8  3.49  
2   0.8     1.8  2.77  
3   0.3     1.3  2.31  
4     1     2.1   3.1  
5   0.9     1.4  3.19  
6   1.6     2.6  3.59  
7   2.5     2.7     4  
8   1.5     1.4  3.51  
9   2.1     1.1  3.57  
10  0.7     0.2  3.43  
11  5.1     3.2  4.39  
12  3.8     1.5  3.75  
13  3.1     2.3  4.04

In [2]:
expenditure_standard = expenditure_standard.drop(index = [0, 1]).reset_index(drop=True)

expenditure_standard

NameError: ignored

## 1인당 지출 경비 - 전체 비용
- 해당파일에 대한 전처리

In [ ]:
total_cost_2018 = pd.read_excel('/content/drive/MyDrive/2018 1인당 지출 경비 - 전체.xlsx')
total_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3       Unnamed: 4  \
0                   NaN           NaN        사례수    10만원 미만  10만원 이상-30만원 미만   
1                  [전체]           NaN       5884        0.2             12.6   
2                    월별           01월        438        1.3             19.5   
3                    월별           02월        414        0.6             25.8   
4                    월별           03월        466        0.5             21.4   
5                    월별           04월        520          0             11.9   
6                    월별           05월        510          0              7.8   
7                    월별           06월        535          0              9.3   
8                    월별           07월        517          0             12.4   
9                    월별           08월        517          0              3.7   
10                   월별           09월        501          0              6.7   
11                   월별           10월        522          0              8.5   
12                   월별           11월        478          0             14.2   
13                   월별           12월        467        0.2             14.2   
14                  거주지         서울특별시       1424        0.2             13.2   
15                  거주지         부산광역시        697        0.2              9.6   
16                  거주지         대구광역시        512          0             11.5   
17                  거주지         인천광역시        353          0             13.7   
18                  거주지         광주광역시        205          0             15.4   
19                  거주지         대전광역시        168          0               11   
20                  거주지         울산광역시        176          0              9.6   
21                  거주지         세종자치시         32          0                4   
22                  거주지           경기도       1226        0.1             13.8   
23                  거주지           강원도         95        1.3              6.9   
24                  거주지           충청도        239        0.5             15.4   
25                  거주지           전라도        361          0             13.4   
26                  거주지           경상도        385          1             12.4   
27                  거주지  해외거주 등 기타 지역         11          0                0   
28                   성별            남자       2456        0.2             12.6   
29                   성별            여자       3428        0.2             12.5   
30                   연령     15세 - 20세        103        2.4             29.4   
31                   연령     21세 - 30세       1388        0.2             14.9   
32                   연령     31세 - 40세       1491          0             13.8   
33                   연령     41세 - 50세       1202        0.2             11.9   
34                   연령     51세 - 60세       1267        0.2              8.4   
35                   연령        60세 이상        433        0.3             10.4   
36                 방문목적     휴가 및 순수여행       5329        0.1             11.1   
37                 방문목적        비즈니스여행        359        0.5             21.5   
38                 방문목적          기타여행        196        1.3             34.7   
39                 여행형태          개별여행       5884        0.2             12.6   
40                 방문횟수            1회       1833        0.4             12.2   
41                 방문횟수            2회       2370        0.2             10.6   
42                 방문횟수            3회        773          0             15.4   
43                 방문횟수         4회 이상        908        0.2             15.9   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN              NaN   

         Unnamed: 5        Unnamed: 6         Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   30만원 이상-50만원 미만  50만원 이상-100만원 미만  100만원 이상-150만원 미만   150만원 이상       [평균]  
1              48.6                35                2.4        1.3  508759.39  
2              36.5              34.8                4.1        3.9  563737.56  
3            

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '10만원 미만',
           'Unnamed: 4' : '10만원 이상-30만원 미만',
           'Unnamed: 5' : '30만원 이상-50만원 미만',
           'Unnamed: 6' : '50만원 이상-100만원 미만',
           'Unnamed: 7' : '100만원 이상-150만원 미만',
           'Unnamed: 8' : '150만원 이상',
           'Unnamed: 9' : '평균'}

total_cost_2018 = total_cost_2018.rename(columns = columns)
total_cost_2018


분석 카테고리       세부 카테고리   사례수  10만원 미만  10만원 이상-30만원 미만  \
0                   NaN           NaN   사례수  10만원 미만  10만원 이상-30만원 미만   
1                  [전체]           NaN  5884      0.2             12.6   
2                    월별           01월   438      1.3             19.5   
3                    월별           02월   414      0.6             25.8   
4                    월별           03월   466      0.5             21.4   
5                    월별           04월   520        0             11.9   
6                    월별           05월   510        0              7.8   
7                    월별           06월   535        0              9.3   
8                    월별           07월   517        0             12.4   
9                    월별           08월   517        0              3.7   
10                   월별           09월   501        0              6.7   
11                   월별           10월   522        0              8.5   
12                   월별           11월   478        0             14.2   
13                   월별           12월   467      0.2             14.2   
14                  거주지         서울특별시  1424      0.2             13.2   
15                  거주지         부산광역시   697      0.2              9.6   
16                  거주지         대구광역시   512        0             11.5   
17                  거주지         인천광역시   353        0             13.7   
18                  거주지         광주광역시   205        0             15.4   
19                  거주지         대전광역시   168        0               11   
20                  거주지         울산광역시   176        0              9.6   
21                  거주지         세종자치시    32        0                4   
22                  거주지           경기도  1226      0.1             13.8   
23                  거주지           강원도    95      1.3              6.9   
24                  거주지           충청도   239      0.5             15.4   
25                  거주지           전라도   361        0             13.4   
26                  거주지           경상도   385        1             12.4   
27                  거주지  해외거주 등 기타 지역    11        0                0   
28                   성별            남자  2456      0.2             12.6   
29                   성별            여자  3428      0.2             12.5   
30                   연령     15세 - 20세   103      2.4             29.4   
31                   연령     21세 - 30세  1388      0.2             14.9   
32                   연령     31세 - 40세  1491        0             13.8   
33                   연령     41세 - 50세  1202      0.2             11.9   
34                   연령     51세 - 60세  1267      0.2              8.4   
35                   연령        60세 이상   433      0.3             10.4   
36                 방문목적     휴가 및 순수여행  5329      0.1             11.1   
37                 방문목적        비즈니스여행   359      0.5             21.5   
38                 방문목적          기타여행   196      1.3             34.7   
39                 여행형태          개별여행  5884      0.2             12.6   
40                 방문횟수            1회  1833      0.4             12.2   
41                 방문횟수            2회  2370      0.2             10.6   
42                 방문횟수            3회   773        0             15.4   
43                 방문횟수         4회 이상   908      0.2             15.9   
44  ♠BASE : 13번의 1번 응답자           NaN   NaN      NaN              NaN   

    30만원 이상-50만원 미만  50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상         평균  
0   30만원 이상-50만원 미만  50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상       [평균]  
1              48.6                35                2.4       1.3  508759.39  
2              36.5              34.8                4.1       3.9  563737.56  
3              41.4              27.4                3.1       1.7   481899.1  
4              47.4                27                2.4       1.2  463494.31  
5              52.1                34                1.1       0.8  485475.92  
6              44.6              43.6                2.7       1.2     561425  
7              43

In [ ]:
total_cost_2018 = total_cost_2018.drop(index = [0, 1]).reset_index(drop=True)

total_cost_2018

분석 카테고리       세부 카테고리   사례수 10만원 미만 10만원 이상-30만원 미만  \
0                    월별           01월   438     1.3            19.5   
1                    월별           02월   414     0.6            25.8   
2                    월별           03월   466     0.5            21.4   
3                    월별           04월   520       0            11.9   
4                    월별           05월   510       0             7.8   
5                    월별           06월   535       0             9.3   
6                    월별           07월   517       0            12.4   
7                    월별           08월   517       0             3.7   
8                    월별           09월   501       0             6.7   
9                    월별           10월   522       0             8.5   
10                   월별           11월   478       0            14.2   
11                   월별           12월   467     0.2            14.2   
12                  거주지         서울특별시  1424     0.2            13.2   
13                  거주지         부산광역시   697     0.2             9.6   
14                  거주지         대구광역시   512       0            11.5   
15                  거주지         인천광역시   353       0            13.7   
16                  거주지         광주광역시   205       0            15.4   
17                  거주지         대전광역시   168       0              11   
18                  거주지         울산광역시   176       0             9.6   
19                  거주지         세종자치시    32       0               4   
20                  거주지           경기도  1226     0.1            13.8   
21                  거주지           강원도    95     1.3             6.9   
22                  거주지           충청도   239     0.5            15.4   
23                  거주지           전라도   361       0            13.4   
24                  거주지           경상도   385       1            12.4   
25                  거주지  해외거주 등 기타 지역    11       0               0   
26                   성별            남자  2456     0.2            12.6   
27                   성별            여자  3428     0.2            12.5   
28                   연령     15세 - 20세   103     2.4            29.4   
29                   연령     21세 - 30세  1388     0.2            14.9   
30                   연령     31세 - 40세  1491       0            13.8   
31                   연령     41세 - 50세  1202     0.2            11.9   
32                   연령     51세 - 60세  1267     0.2             8.4   
33                   연령        60세 이상   433     0.3            10.4   
34                 방문목적     휴가 및 순수여행  5329     0.1            11.1   
35                 방문목적        비즈니스여행   359     0.5            21.5   
36                 방문목적          기타여행   196     1.3            34.7   
37                 여행형태          개별여행  5884     0.2            12.6   
38                 방문횟수            1회  1833     0.4            12.2   
39                 방문횟수            2회  2370     0.2            10.6   
40                 방문횟수            3회   773       0            15.4   
41                 방문횟수         4회 이상   908     0.2            15.9   
42  ♠BASE : 13번의 1번 응답자           NaN   NaN     NaN             NaN   

   30만원 이상-50만원 미만 50만원 이상-100만원 미만 100만원 이상-150만원 미만 150만원 이상         평균  
0             36.5             34.8               4.1      3.9  563737.56  
1             41.4             27.4               3.1      1.7   481899.1  
2             47.4               27               2.4      1.2  463494.31  
3             52.1               34               1.1      0.8  485475.92  
4             44.6             43.6               2.7      1.2     561425  
5             43.2             43.3               2.6      1.5  539357.97  
6             51.2             33.4               2.6      0.5  494075.57  
7             40.9             49.6               4.4      1.5  578370.98  
8             49.5             40.4                 3      0.4  515749.92  
9               56             32.7                 1      1.9  502549.06  
10            56.3               27               2.1  

In [ ]:
total_cost_2018 = total_cost_2018.drop(index = range(12, 28)).reset_index(drop=True)

total_cost_2018

분석 카테고리    세부 카테고리   사례수 10만원 미만 10만원 이상-30만원 미만  \
0                    월별        01월   438     1.3            19.5   
1                    월별        02월   414     0.6            25.8   
2                    월별        03월   466     0.5            21.4   
3                    월별        04월   520       0            11.9   
4                    월별        05월   510       0             7.8   
5                    월별        06월   535       0             9.3   
6                    월별        07월   517       0            12.4   
7                    월별        08월   517       0             3.7   
8                    월별        09월   501       0             6.7   
9                    월별        10월   522       0             8.5   
10                   월별        11월   478       0            14.2   
11                   월별        12월   467     0.2            14.2   
12                   연령  15세 - 20세   103     2.4            29.4   
13                   연령  21세 - 30세  1388     0.2            14.9   
14                   연령  31세 - 40세  1491       0            13.8   
15                   연령  41세 - 50세  1202     0.2            11.9   
16                   연령  51세 - 60세  1267     0.2             8.4   
17                   연령     60세 이상   433     0.3            10.4   
18                 방문목적  휴가 및 순수여행  5329     0.1            11.1   
19                 방문목적     비즈니스여행   359     0.5            21.5   
20                 방문목적       기타여행   196     1.3            34.7   
21                 여행형태       개별여행  5884     0.2            12.6   
22                 방문횟수         1회  1833     0.4            12.2   
23                 방문횟수         2회  2370     0.2            10.6   
24                 방문횟수         3회   773       0            15.4   
25                 방문횟수      4회 이상   908     0.2            15.9   
26  ♠BASE : 13번의 1번 응답자        NaN   NaN     NaN             NaN   

   30만원 이상-50만원 미만 50만원 이상-100만원 미만 100만원 이상-150만원 미만 150만원 이상         평균  
0             36.5             34.8               4.1      3.9  563737.56  
1             41.4             27.4               3.1      1.7   481899.1  
2             47.4               27               2.4      1.2  463494.31  
3             52.1               34               1.1      0.8  485475.92  
4             44.6             43.6               2.7      1.2     561425  
5             43.2             43.3               2.6      1.5  539357.97  
6             51.2             33.4               2.6      0.5  494075.57  
7             40.9             49.6               4.4      1.5  578370.98  
8             49.5             40.4                 3      0.4  515749.92  
9               56             32.7                 1      1.9  502549.06  
10            56.3               27               2.1      0.3  474495.68  
11            62.7               22               0.2      0.6  433259.34  
12              46             17.4               2.4      2.4  425000.21  
13            56.3             26.9                 1      0.7  458519.22  
14            46.8             34.7               3.3      1.4  514455.46  
15            46.8             37.1               2.6      1.3  522569.74  
16            44.6             42.2               2.9      1.7  546813.59  
17            47.5             38.8                 2        1   520510.1  
18              50             35.4               2.3        1  505553.23  
19            37.8             32.3               4.7      3.2  548112.45  
20            31.1             26.4               2.7      3.8  523840.85  
21            48.6               35               2.4      1.3  508759.39  
22            50.7             34.5               1.6      0.7  489298.95  
23            52.9             33.5               2.1      0.8   492395.4  
24            43.6             36.2               3.1      1.6  519012.45  
25            37.5             38.8               4.3      3.4  582000.26  
26             NaN              NaN               NaN      

In [ ]:
total_cost_2018 = total_cost_2018.drop(index = range(18, 27)).reset_index(drop=True)

total_cost_2018

분석 카테고리    세부 카테고리   사례수 10만원 미만 10만원 이상-30만원 미만 30만원 이상-50만원 미만  \
0       월별        01월   438     1.3            19.5            36.5   
1       월별        02월   414     0.6            25.8            41.4   
2       월별        03월   466     0.5            21.4            47.4   
3       월별        04월   520       0            11.9            52.1   
4       월별        05월   510       0             7.8            44.6   
5       월별        06월   535       0             9.3            43.2   
6       월별        07월   517       0            12.4            51.2   
7       월별        08월   517       0             3.7            40.9   
8       월별        09월   501       0             6.7            49.5   
9       월별        10월   522       0             8.5              56   
10      월별        11월   478       0            14.2            56.3   
11      월별        12월   467     0.2            14.2            62.7   
12      연령  15세 - 20세   103     2.4            29.4              46   
13      연령  21세 - 30세  1388     0.2            14.9            56.3   
14      연령  31세 - 40세  1491       0            13.8            46.8   
15      연령  41세 - 50세  1202     0.2            11.9            46.8   
16      연령  51세 - 60세  1267     0.2             8.4            44.6   
17      연령     60세 이상   433     0.3            10.4            47.5   

   50만원 이상-100만원 미만 100만원 이상-150만원 미만 150만원 이상         평균  
0              34.8               4.1      3.9  563737.56  
1              27.4               3.1      1.7   481899.1  
2                27               2.4      1.2  463494.31  
3                34               1.1      0.8  485475.92  
4              43.6               2.7      1.2     561425  
5              43.3               2.6      1.5  539357.97  
6              33.4               2.6      0.5  494075.57  
7              49.6               4.4      1.5  578370.98  
8              40.4                 3      0.4  515749.92  
9              32.7                 1      1.9  502549.06  
10               27               2.1      0.3  474495.68  
11               22               0.2      0.6  433259.34  
12             17.4               2.4      2.4  425000.21  
13             26.9                 1      0.7  458519.22  
14             34.7               3.3      1.4  514455.46  
15             37.1               2.6      1.3  522569.74  
16             42.2               2.9      1.7  546813.59  
17             38.8                 2        1   520510.1

1) 월별 소비 패턴:

- 10만원 미만 범위의 소비자 수는 월별로 다소 변동이 있지만, 특히 1월과 2월에 낮은 수치를 보이고, 3월부터 5월까지 상대적으로 높아짐.

- 6월부터 8월까지는 50만원 이상-100만원 미만 범위의 비율이 높아지는 특징을 보임.

- 12월에는 10만원 이상-30만원 미만 범위와 50만원 이상-100만원 미만 범위의 비율이 높아지며, 10만원 미만 범위는 상대적으로 낮아짐.

2) 평균 금액:

- 가장 높은 평균 소비 금액은 7월(578,371원)이고, 그 다음으로 1월(563,738원)

- 가장 낮은 평균 소비 금액은 12월(433,259원)

In [ ]:
total_cost_2018 = total_cost_2018.drop(index = range(12, 18)).reset_index(drop=True)

total_cost_2018

분석 카테고리 세부 카테고리  사례수 10만원 미만 10만원 이상-30만원 미만 30만원 이상-50만원 미만  \
0       월별     01월  438     1.3            19.5            36.5   
1       월별     02월  414     0.6            25.8            41.4   
2       월별     03월  466     0.5            21.4            47.4   
3       월별     04월  520       0            11.9            52.1   
4       월별     05월  510       0             7.8            44.6   
5       월별     06월  535       0             9.3            43.2   
6       월별     07월  517       0            12.4            51.2   
7       월별     08월  517       0             3.7            40.9   
8       월별     09월  501       0             6.7            49.5   
9       월별     10월  522       0             8.5              56   
10      월별     11월  478       0            14.2            56.3   
11      월별     12월  467     0.2            14.2            62.7   

   50만원 이상-100만원 미만 100만원 이상-150만원 미만 150만원 이상         평균  
0              34.8               4.1      3.9  563737.56  
1              27.4               3.1      1.7   481899.1  
2                27               2.4      1.2  463494.31  
3                34               1.1      0.8  485475.92  
4              43.6               2.7      1.2     561425  
5              43.3               2.6      1.5  539357.97  
6              33.4               2.6      0.5  494075.57  
7              49.6               4.4      1.5  578370.98  
8              40.4                 3      0.4  515749.92  
9              32.7                 1      1.9  502549.06  
10               27               2.1      0.3  474495.68  
11               22               0.2      0.6  433259.34

## 1인당 지출 경비 - 숙박 비용
- 해당 파일에 대한 전처리


In [ ]:
accommodation_cost_2018 = pd.read_excel('/content/drive/MyDrive/2018 1인당 지출 경비 - 숙박비.xlsx')
accommodation_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4  \
0                   NaN           NaN        사례수     5만원 미만  5만원 이상-7만원 미만   
1                  [전체]           NaN       5884       15.3           24.4   
2                    월별           01월        438       11.2           15.1   
3                    월별           02월        414       16.5           14.5   
4                    월별           03월        466       20.8           22.7   
5                    월별           04월        520       19.8           33.7   
6                    월별           05월        510       11.6           25.9   
7                    월별           06월        535       17.4             31   
8                    월별           07월        517         12           20.7   
9                    월별           08월        517          9           15.4   
10                   월별           09월        501        8.4           20.5   
11                   월별           10월        522       17.9             24   
12                   월별           11월        478       22.8             30   
13                   월별           12월        467       16.4           36.4   
14                  거주지         서울특별시       1424       13.6           21.2   
15                  거주지         부산광역시        697       12.6           23.7   
16                  거주지         대구광역시        512       16.6           22.5   
17                  거주지         인천광역시        353       16.1             29   
18                  거주지         광주광역시        205       19.4           26.7   
19                  거주지         대전광역시        168       13.3           22.1   
20                  거주지         울산광역시        176        9.9           31.7   
21                  거주지         세종자치시         32        7.4           15.3   
22                  거주지           경기도       1226       16.9           26.4   
23                  거주지           강원도         95       11.6             23   
24                  거주지           충청도        239       16.6           28.3   
25                  거주지           전라도        361       19.7           21.6   
26                  거주지           경상도        385       17.1           26.7   
27                  거주지  해외거주 등 기타 지역         11          0           10.1   
28                   성별            남자       2456       12.9           23.6   
29                   성별            여자       3428         17           24.9   
30                   연령     15세 - 20세        103       22.5           17.7   
31                   연령     21세 - 30세       1388         16             26   
32                   연령     31세 - 40세       1491       13.4           19.6   
33                   연령     41세 - 50세       1202         15           26.6   
34                   연령     51세 - 60세       1267       15.5           26.4   
35                   연령        60세 이상        433       17.8           24.6   
36                 방문목적     휴가 및 순수여행       5329       15.6             25   
37                 방문목적        비즈니스여행        359       10.1           20.4   
38                 방문목적          기타여행        196       15.6           14.5   
39                 여행형태          개별여행       5884       15.3           24.4   
40                 방문횟수            1회       1833         16           27.1   
41                 방문횟수            2회       2370       15.5           25.8   
42                 방문횟수            3회        773       16.2           22.4   
43                 방문횟수         4회 이상        908       12.5           16.8   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN            NaN   

        Unnamed: 5       Unnamed: 6       Unnamed: 7 Unnamed: 8 Unnamed: 9  \
0   7만원 이상-10만원 미만  10만원 이상-20만원 미만  20만원 이상-30만원 미만    30만원 이상       경험없음   
1             19.9               27              5.2        3.2        5.1   
2             11.4             33.4              8.6        6.9       13.5   
3             16.3             28.1              7.8          5       11.6   
4             16.5             21.9    

In [ ]:
accommodation_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4  \
0                   NaN           NaN        사례수     5만원 미만  5만원 이상-7만원 미만   
1                  [전체]           NaN       5884       15.3           24.4   
2                    월별           01월        438       11.2           15.1   
3                    월별           02월        414       16.5           14.5   
4                    월별           03월        466       20.8           22.7   
5                    월별           04월        520       19.8           33.7   
6                    월별           05월        510       11.6           25.9   
7                    월별           06월        535       17.4             31   
8                    월별           07월        517         12           20.7   
9                    월별           08월        517          9           15.4   
10                   월별           09월        501        8.4           20.5   
11                   월별           10월        522       17.9             24   
12                   월별           11월        478       22.8             30   
13                   월별           12월        467       16.4           36.4   
14                  거주지         서울특별시       1424       13.6           21.2   
15                  거주지         부산광역시        697       12.6           23.7   
16                  거주지         대구광역시        512       16.6           22.5   
17                  거주지         인천광역시        353       16.1             29   
18                  거주지         광주광역시        205       19.4           26.7   
19                  거주지         대전광역시        168       13.3           22.1   
20                  거주지         울산광역시        176        9.9           31.7   
21                  거주지         세종자치시         32        7.4           15.3   
22                  거주지           경기도       1226       16.9           26.4   
23                  거주지           강원도         95       11.6             23   
24                  거주지           충청도        239       16.6           28.3   
25                  거주지           전라도        361       19.7           21.6   
26                  거주지           경상도        385       17.1           26.7   
27                  거주지  해외거주 등 기타 지역         11          0           10.1   
28                   성별            남자       2456       12.9           23.6   
29                   성별            여자       3428         17           24.9   
30                   연령     15세 - 20세        103       22.5           17.7   
31                   연령     21세 - 30세       1388         16             26   
32                   연령     31세 - 40세       1491       13.4           19.6   
33                   연령     41세 - 50세       1202         15           26.6   
34                   연령     51세 - 60세       1267       15.5           26.4   
35                   연령        60세 이상        433       17.8           24.6   
36                 방문목적     휴가 및 순수여행       5329       15.6             25   
37                 방문목적        비즈니스여행        359       10.1           20.4   
38                 방문목적          기타여행        196       15.6           14.5   
39                 여행형태          개별여행       5884       15.3           24.4   
40                 방문횟수            1회       1833         16           27.1   
41                 방문횟수            2회       2370       15.5           25.8   
42                 방문횟수            3회        773       16.2           22.4   
43                 방문횟수         4회 이상        908       12.5           16.8   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN            NaN   

        Unnamed: 5       Unnamed: 6       Unnamed: 7 Unnamed: 8 Unnamed: 9  \
0   7만원 이상-10만원 미만  10만원 이상-20만원 미만  20만원 이상-30만원 미만    30만원 이상       경험없음   
1             19.9               27              5.2        3.2        5.1   
2             11.4             33.4              8.6        6.9       13.5   
3             16.3             28.1              7.8          5       11.6   
4             16.5             21.9    

In [ ]:
accommodation_cost_2018 = accommodation_cost_2018.drop(index = range (14,45)).reset_index(drop = True)

accommodation_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4      Unnamed: 5  \
0    NaN        NaN        사례수     5만원 미만  5만원 이상-7만원 미만  7만원 이상-10만원 미만   
1   [전체]        NaN       5884       15.3           24.4            19.9   
2     월별        01월        438       11.2           15.1            11.4   
3     월별        02월        414       16.5           14.5            16.3   
4     월별        03월        466       20.8           22.7            16.5   
5     월별        04월        520       19.8           33.7            15.4   
6     월별        05월        510       11.6           25.9            22.6   
7     월별        06월        535       17.4             31            19.5   
8     월별        07월        517         12           20.7            20.7   
9     월별        08월        517          9           15.4            21.9   
10    월별        09월        501        8.4           20.5            29.3   
11    월별        10월        522       17.9             24            25.2   
12    월별        11월        478       22.8             30            17.3   
13    월별        12월        467       16.4           36.4            20.2   

         Unnamed: 6       Unnamed: 7 Unnamed: 8 Unnamed: 9 Unnamed: 10  
0   10만원 이상-20만원 미만  20만원 이상-30만원 미만    30만원 이상       경험없음        [평균]  
1                27              5.2        3.2        5.1     95507.9  
2              33.4              8.6        6.9       13.5   124757.26  
3              28.1              7.8          5       11.6   105754.63  
4              21.9              3.5        3.3       11.3    83404.74  
5              21.2              3.6        2.6        3.6    81586.72  
6              31.1              3.2        2.5        3.1    91445.22  
7              20.4              3.9        2.8          5    85866.59  
8              32.4              8.1          3        3.1   104168.02  
9                36              8.3        5.8        3.6   117563.07  
10               33              4.4        1.9        2.6     96879.2  
11             24.8              3.8        2.8        1.5    94885.76  
12             21.2              4.7          1        3.1    83596.91  
13             20.4              3.5        1.1          2    79441.69

In [ ]:
accommodation_cost_2018 = accommodation_cost_2018.drop(index = [0, 1]).reset_index(drop=True)

accommodation_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0     월별        01월        438       11.2       15.1       11.4       33.4   
1     월별        02월        414       16.5       14.5       16.3       28.1   
2     월별        03월        466       20.8       22.7       16.5       21.9   
3     월별        04월        520       19.8       33.7       15.4       21.2   
4     월별        05월        510       11.6       25.9       22.6       31.1   
5     월별        06월        535       17.4         31       19.5       20.4   
6     월별        07월        517         12       20.7       20.7       32.4   
7     월별        08월        517          9       15.4       21.9         36   
8     월별        09월        501        8.4       20.5       29.3         33   
9     월별        10월        522       17.9         24       25.2       24.8   
10    월별        11월        478       22.8         30       17.3       21.2   
11    월별        12월        467       16.4       36.4       20.2       20.4   

   Unnamed: 7 Unnamed: 8 Unnamed: 9 Unnamed: 10  
0         8.6        6.9       13.5   124757.26  
1         7.8          5       11.6   105754.63  
2         3.5        3.3       11.3    83404.74  
3         3.6        2.6        3.6    81586.72  
4         3.2        2.5        3.1    91445.22  
5         3.9        2.8          5    85866.59  
6         8.1          3        3.1   104168.02  
7         8.3        5.8        3.6   117563.07  
8         4.4        1.9        2.6     96879.2  
9         3.8        2.8        1.5    94885.76  
10        4.7          1        3.1    83596.91  
11        3.5        1.1          2    79441.69

1) 월별 소비 패턴:

- 대체로 10만원 이상-20만원 미만 범위의 비율이 높아 보이며, 특히 4월과 5월에서 이 범위의 비율이 상대적으로 높음.

- 7월과 8월에는 20만원 이상-30만원 미만 범위의 비율이 높아지며, 11월과 12월에는 5만원 이상-7만원 미만 범위의 비율이 높아짐.

- 9월과 10월에는 다양한 범위의 비율이 균형적으로 분포됨.

2) 평균 금액:

- 8월(117,563.07원)과 1월(124,757.26원)이 가장 높은 평균 소비 금액을 보임.

- 12월(79,441.69원)가 가장 낮은 평균 소비 금액을 보임.

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '5만원 미만',
           'Unnamed: 4' : '5만원 이상-7만원 미만',
           'Unnamed: 5' : '7만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상-30만원 미만',
           'Unnamed: 8' : '30만원 이상',
           'Unnamed: 9' : '경험없음',
           'Unnamed: 10' : '평균'}

accommodation_cost_2018 = accommodation_cost_2018.rename(columns = columns)
accommodation_cost_2018

분석 카테고리 세부 카테고리  사례수 5만원 미만 5만원 이상-7만원 미만 7만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0       월별     01월  438   11.2          15.1           11.4            33.4   
1       월별     02월  414   16.5          14.5           16.3            28.1   
2       월별     03월  466   20.8          22.7           16.5            21.9   
3       월별     04월  520   19.8          33.7           15.4            21.2   
4       월별     05월  510   11.6          25.9           22.6            31.1   
5       월별     06월  535   17.4            31           19.5            20.4   
6       월별     07월  517     12          20.7           20.7            32.4   
7       월별     08월  517      9          15.4           21.9              36   
8       월별     09월  501    8.4          20.5           29.3              33   
9       월별     10월  522   17.9            24           25.2            24.8   
10      월별     11월  478   22.8            30           17.3            21.2   
11      월별     12월  467   16.4          36.4           20.2            20.4   

   20만원 이상-30만원 미만 30만원 이상  경험없음         평균  
0              8.6     6.9  13.5  124757.26  
1              7.8       5  11.6  105754.63  
2              3.5     3.3  11.3   83404.74  
3              3.6     2.6   3.6   81586.72  
4              3.2     2.5   3.1   91445.22  
5              3.9     2.8     5   85866.59  
6              8.1       3   3.1  104168.02  
7              8.3     5.8   3.6  117563.07  
8              4.4     1.9   2.6    96879.2  
9              3.8     2.8   1.5   94885.76  
10             4.7       1   3.1   83596.91  
11             3.5     1.1     2   79441.69

## 1 인당 지출 경비 - 식음료 비용
- 해당 파일에 대한 전처리

In [ ]:
food_cost_2018 = pd.read_excel('/content/drive/MyDrive/2018 1인당 지출 경비 - 식음료비.xlsx')
food_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4  \
0                   NaN           NaN        사례수     3만원 미만  3만원이상-5만원 미만   
1                  [전체]           NaN       5884        2.3           1.5   
2                    월별           01월        438        5.3           3.5   
3                    월별           02월        414        7.8           5.8   
4                    월별           03월        466        7.7           5.1   
5                    월별           04월        520        1.9           1.3   
6                    월별           05월        510        0.5             1   
7                    월별           06월        535        2.1           0.7   
8                    월별           07월        517        1.4           0.6   
9                    월별           08월        517        0.5           0.5   
10                   월별           09월        501          1             0   
11                   월별           10월        522        0.5           0.2   
12                   월별           11월        478        0.5           0.7   
13                   월별           12월        467        0.5           0.5   
14                  거주지         서울특별시       1424        2.3           1.3   
15                  거주지         부산광역시        697        1.7           0.9   
16                  거주지         대구광역시        512        2.3             1   
17                  거주지         인천광역시        353        1.2           1.8   
18                  거주지         광주광역시        205        3.2             3   
19                  거주지         대전광역시        168        3.7           0.8   
20                  거주지         울산광역시        176        1.3           0.6   
21                  거주지         세종자치시         32          0             0   
22                  거주지           경기도       1226        1.9             2   
23                  거주지           강원도         95        7.5           2.5   
24                  거주지           충청도        239        3.6           2.3   
25                  거주지           전라도        361        2.2           0.9   
26                  거주지           경상도        385        3.9           2.4   
27                  거주지  해외거주 등 기타 지역         11          0          11.4   
28                   성별            남자       2456        2.7           1.4   
29                   성별            여자       3428        2.1           1.6   
30                   연령     15세 - 20세        103          6          10.6   
31                   연령     21세 - 30세       1388        3.2             2   
32                   연령     31세 - 40세       1491        1.9           1.3   
33                   연령     41세 - 50세       1202        2.2           1.6   
34                   연령     51세 - 60세       1267          2           0.8   
35                   연령        60세 이상        433        1.5             1   
36                 방문목적     휴가 및 순수여행       5329        1.6           1.3   
37                 방문목적        비즈니스여행        359          9           3.3   
38                 방문목적          기타여행        196       10.9           4.5   
39                 여행형태          개별여행       5884        2.3           1.5   
40                 방문횟수            1회       1833        2.3           1.8   
41                 방문횟수            2회       2370        1.7             1   
42                 방문횟수            3회        773          2           1.9   
43                 방문횟수         4회 이상        908        4.4           2.3   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN           NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   5만원 이상-10만원 미만  10만원 이상-20만원 미만    20만원 이상       경험없음       [평균]  
1             21.4             55.9       17.5        1.2  138575.18  
2             26.4             39.2       22.7          3  150879.85  
3             25.6             39.4       16.4        4.9  121012.57  
4               27             45.4       12.1        2.7  110899.17  
5             26.8             54.4       14.4   

In [ ]:
food_cost_2018 = food_cost_2018.drop(index = range(14, 45)).reset_index(drop=True)

food_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4      Unnamed: 5  \
0    NaN        NaN        사례수     3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만   
1   [전체]        NaN       5884        2.3           1.5            21.4   
2     월별        01월        438        5.3           3.5            26.4   
3     월별        02월        414        7.8           5.8            25.6   
4     월별        03월        466        7.7           5.1              27   
5     월별        04월        520        1.9           1.3            26.8   
6     월별        05월        510        0.5             1            22.6   
7     월별        06월        535        2.1           0.7            20.5   
8     월별        07월        517        1.4           0.6            14.3   
9     월별        08월        517        0.5           0.5            11.4   
10    월별        09월        501          1             0              14   
11    월별        10월        522        0.5           0.2            18.1   
12    월별        11월        478        0.5           0.7            24.5   
13    월별        12월        467        0.5           0.5            28.9   

         Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   10만원 이상-20만원 미만    20만원 이상       경험없음       [평균]  
1              55.9       17.5        1.2  138575.18  
2              39.2       22.7          3  150879.85  
3              39.4       16.4        4.9  121012.57  
4              45.4       12.1        2.7  110899.17  
5              54.4       14.4        1.2  129662.94  
6              55.6       19.4          1  144911.62  
7              57.4       18.9        0.3   141012.4  
8              60.5       22.4        0.8  156353.96  
9              58.5       29.2          0  167681.77  
10             63.4       20.8        0.8  145992.95  
11             66.4         14        0.7  137954.86  
12             63.9       10.4          0     130043  
13             61.6        8.5          0  120002.96

In [ ]:
food_cost_2018 = food_cost_2018.drop(index = [0, 1]).reset_index(drop=True)

food_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0     월별        01월        438        5.3        3.5       26.4       39.2   
1     월별        02월        414        7.8        5.8       25.6       39.4   
2     월별        03월        466        7.7        5.1         27       45.4   
3     월별        04월        520        1.9        1.3       26.8       54.4   
4     월별        05월        510        0.5          1       22.6       55.6   
5     월별        06월        535        2.1        0.7       20.5       57.4   
6     월별        07월        517        1.4        0.6       14.3       60.5   
7     월별        08월        517        0.5        0.5       11.4       58.5   
8     월별        09월        501          1          0         14       63.4   
9     월별        10월        522        0.5        0.2       18.1       66.4   
10    월별        11월        478        0.5        0.7       24.5       63.9   
11    월별        12월        467        0.5        0.5       28.9       61.6   

   Unnamed: 7 Unnamed: 8 Unnamed: 9  
0        22.7          3  150879.85  
1        16.4        4.9  121012.57  
2        12.1        2.7  110899.17  
3        14.4        1.2  129662.94  
4        19.4          1  144911.62  
5        18.9        0.3   141012.4  
6        22.4        0.8  156353.96  
7        29.2          0  167681.77  
8        20.8        0.8  145992.95  
9          14        0.7  137954.86  
10       10.4          0     130043  
11        8.5          0  120002.96

1) 월별 소비 패턴:

- 대체로 10만원 이상-20만원 미만 범위의 비율이 높아 보이며, 특히 8월(58.5%)에서 이 범위의 비율이 가장 높음.

- 3만원 미만 범위와 3만원 이상-5만원 미만 범위의 비율은 점차 감소하고, 10만원 이상-20만원 미만 범위와 20만원 이상 범위의 비율이 증가하는 경향을 보임.

- 11월(63.9%)과 12월(61.6%)에는 10만원 이상-20만원 미만 범위와 20만원 이상 범위의 비율이 높음.

2) 평균 금액:

- 8월(167,681.77원)과 7월(156,353.96원)이 가장 높은 평균 소비 금액.

- 12월(120,002.96원)가 가장 낮은 평균 소비 금액을 보임.

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '3만원 미만',
           'Unnamed: 4' : '3만원이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상',
           'Unnamed: 8' : '경험없음',
           'Unnamed: 9' : '평균'}

food_cost_2018 = food_cost_2018.rename(columns = columns)
food_cost_2018

분석 카테고리 세부 카테고리  사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0       월별     01월  438    5.3          3.5           26.4            39.2   
1       월별     02월  414    7.8          5.8           25.6            39.4   
2       월별     03월  466    7.7          5.1             27            45.4   
3       월별     04월  520    1.9          1.3           26.8            54.4   
4       월별     05월  510    0.5            1           22.6            55.6   
5       월별     06월  535    2.1          0.7           20.5            57.4   
6       월별     07월  517    1.4          0.6           14.3            60.5   
7       월별     08월  517    0.5          0.5           11.4            58.5   
8       월별     09월  501      1            0             14            63.4   
9       월별     10월  522    0.5          0.2           18.1            66.4   
10      월별     11월  478    0.5          0.7           24.5            63.9   
11      월별     12월  467    0.5          0.5           28.9            61.6   

   20만원 이상 경험없음         평균  
0     22.7    3  150879.85  
1     16.4  4.9  121012.57  
2     12.1  2.7  110899.17  
3     14.4  1.2  129662.94  
4     19.4    1  144911.62  
5     18.9  0.3   141012.4  
6     22.4  0.8  156353.96  
7     29.2    0  167681.77  
8     20.8  0.8  145992.95  
9       14  0.7  137954.86  
10    10.4    0     130043  
11     8.5    0  120002.96

## 1인당 지출 경비 - 항공/선박 이용 비용
- 해당 파일에 대한 전처리

In [ ]:
transportation_cost_2018 = pd.read_excel('/content/drive/MyDrive/2018 1인당 지출 경비 - 항공료.xlsx')
transportation_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4  \
0                   NaN           NaN        사례수     3만원 미만  3만원이상-5만원 미만   
1                  [전체]           NaN       5884          2           1.3   
2                    월별           01월        438        3.7           5.1   
3                    월별           02월        414        2.4             4   
4                    월별           03월        466        1.6           1.4   
5                    월별           04월        520          2           0.5   
6                    월별           05월        510        1.6             1   
7                    월별           06월        535        1.9           0.7   
8                    월별           07월        517        3.8           1.1   
9                    월별           08월        517        1.4             0   
10                   월별           09월        501        0.5           0.3   
11                   월별           10월        522        3.1           1.5   
12                   월별           11월        478        0.7           0.4   
13                   월별           12월        467        1.7           0.8   
14                  거주지         서울특별시       1424        2.7           1.4   
15                  거주지         부산광역시        697        1.9           0.5   
16                  거주지         대구광역시        512        1.7             1   
17                  거주지         인천광역시        353        1.1           1.6   
18                  거주지         광주광역시        205        0.8           2.4   
19                  거주지         대전광역시        168        1.5           0.9   
20                  거주지         울산광역시        176        1.4             3   
21                  거주지         세종자치시         32          0             0   
22                  거주지           경기도       1226        2.3             1   
23                  거주지           강원도         95        6.3           1.4   
24                  거주지           충청도        239        0.6           0.7   
25                  거주지           전라도        361        1.2           2.7   
26                  거주지           경상도        385        1.9           1.4   
27                  거주지  해외거주 등 기타 지역         11       11.1          11.4   
28                   성별            남자       2456        2.1           1.6   
29                   성별            여자       3428          2           1.1   
30                   연령     15세 - 20세        103        7.1           4.4   
31                   연령     21세 - 30세       1388        1.7           1.6   
32                   연령     31세 - 40세       1491        1.9           1.3   
33                   연령     41세 - 50세       1202        1.3           0.9   
34                   연령     51세 - 60세       1267        2.2           1.3   
35                   연령        60세 이상        433        3.5           1.1   
36                 방문목적     휴가 및 순수여행       5329          2           1.3   
37                 방문목적        비즈니스여행        359        2.2           0.9   
38                 방문목적          기타여행        196        3.2           3.2   
39                 여행형태          개별여행       5884          2           1.3   
40                 방문횟수            1회       1833          2           1.2   
41                 방문횟수            2회       2370        2.3           1.3   
42                 방문횟수            3회        773        2.5           1.4   
43                 방문횟수         4회 이상        908          1           1.5   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN           NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8  
0   5만원 이상-10만원 미만  10만원 이상-15만원 미만    15만원 이상       [평균]  
1             21.5             44.5       30.6   125424.3  
2             34.4             35.1       21.7  104332.71  
3             42.4             38.9       12.2    96845.6  
4               26               43       27.9  123391.34  
5             19.9             56.7       20.9  122340.74  
6             11.9             37.2       48.4  152181.

In [ ]:
transportation_cost_2018 = transportation_cost_2018.drop(index = range(14, 45)).reset_index(drop=True)

transportation_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4      Unnamed: 5  \
0    NaN        NaN        사례수     3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만   
1   [전체]        NaN       5884          2           1.3            21.5   
2     월별        01월        438        3.7           5.1            34.4   
3     월별        02월        414        2.4             4            42.4   
4     월별        03월        466        1.6           1.4              26   
5     월별        04월        520          2           0.5            19.9   
6     월별        05월        510        1.6             1            11.9   
7     월별        06월        535        1.9           0.7             5.1   
8     월별        07월        517        3.8           1.1            23.3   
9     월별        08월        517        1.4             0             4.7   
10    월별        09월        501        0.5           0.3            19.1   
11    월별        10월        522        3.1           1.5             8.6   
12    월별        11월        478        0.7           0.4            34.2   
13    월별        12월        467        1.7           0.8            37.7   

         Unnamed: 6 Unnamed: 7 Unnamed: 8  
0   10만원 이상-15만원 미만    15만원 이상       [평균]  
1              44.5       30.6   125424.3  
2              35.1       21.7  104332.71  
3              38.9       12.2    96845.6  
4                43       27.9  123391.34  
5              56.7       20.9  122340.74  
6              37.2       48.4  152181.49  
7              36.2       56.1  154723.37  
8              55.5       16.4  113538.88  
9              29.7       64.2  151071.74  
10             56.7       23.3  119348.38  
11             44.7       42.2  138782.42  
12             44.3       20.3  115986.79  
13             55.4        4.4   99300.75

In [ ]:
transportation_cost_2018 = transportation_cost_2018.drop(index = [0, 1]).reset_index(drop=True)

transportation_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0     월별        01월        438        3.7        5.1       34.4       35.1   
1     월별        02월        414        2.4          4       42.4       38.9   
2     월별        03월        466        1.6        1.4         26         43   
3     월별        04월        520          2        0.5       19.9       56.7   
4     월별        05월        510        1.6          1       11.9       37.2   
5     월별        06월        535        1.9        0.7        5.1       36.2   
6     월별        07월        517        3.8        1.1       23.3       55.5   
7     월별        08월        517        1.4          0        4.7       29.7   
8     월별        09월        501        0.5        0.3       19.1       56.7   
9     월별        10월        522        3.1        1.5        8.6       44.7   
10    월별        11월        478        0.7        0.4       34.2       44.3   
11    월별        12월        467        1.7        0.8       37.7       55.4   

   Unnamed: 7 Unnamed: 8  
0        21.7  104332.71  
1        12.2    96845.6  
2        27.9  123391.34  
3        20.9  122340.74  
4        48.4  152181.49  
5        56.1  154723.37  
6        16.4  113538.88  
7        64.2  151071.74  
8        23.3  119348.38  
9        42.2  138782.42  
10       20.3  115986.79  
11        4.4   99300.75

1) 월별 패턴:

- 월(3월, 4월)에는 3만원 이상에서 10만원 미만의 범주가 비교적 높음.

2) 평균 금액:

- 5월과 6월에 평균 금액이 가장 높고, 12월이 평균 금액이 가장 낮음.

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '3만원 미만',
           'Unnamed: 4' : '3만원이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-15만원 미만',
           'Unnamed: 7' : '15만원 이상',
           'Unnamed: 8' : '평균'}

transportation_cost_2018 = transportation_cost_2018.rename(columns = columns)
transportation_cost_2018

분석 카테고리 세부 카테고리  사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-15만원 미만  \
0       월별     01월  438    3.7          5.1           34.4            35.1   
1       월별     02월  414    2.4            4           42.4            38.9   
2       월별     03월  466    1.6          1.4             26              43   
3       월별     04월  520      2          0.5           19.9            56.7   
4       월별     05월  510    1.6            1           11.9            37.2   
5       월별     06월  535    1.9          0.7            5.1            36.2   
6       월별     07월  517    3.8          1.1           23.3            55.5   
7       월별     08월  517    1.4            0            4.7            29.7   
8       월별     09월  501    0.5          0.3           19.1            56.7   
9       월별     10월  522    3.1          1.5            8.6            44.7   
10      월별     11월  478    0.7          0.4           34.2            44.3   
11      월별     12월  467    1.7          0.8           37.7            55.4   

   15만원 이상         평균  
0     21.7  104332.71  
1     12.2    96845.6  
2     27.9  123391.34  
3     20.9  122340.74  
4     48.4  152181.49  
5     56.1  154723.37  
6     16.4  113538.88  
7     64.2  151071.74  
8     23.3  119348.38  
9     42.2  138782.42  
10    20.3  115986.79  
11     4.4   99300.75

## 1인당 지출 경비 -관광/문화 지출 비용
- 해당 파일에 대한 전처리

In [ ]:
culture_cost_2018 = pd.read_excel('/content/drive/MyDrive/2018 1인당 지출 경비 - 관광비.xlsx')
culture_cost_2018

table    Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4  \
0                   NaN           NaN        사례수     2만원 미만  2만원 이상-5만원 미만   
1                  [전체]           NaN       5884       26.9             32   
2                    월별           01월        438       20.5           13.1   
3                    월별           02월        414       21.4           15.9   
4                    월별           03월        466       28.2           25.6   
5                    월별           04월        520       27.5           39.3   
6                    월별           05월        510       21.1           38.8   
7                    월별           06월        535       24.8           34.9   
8                    월별           07월        517       33.3           25.8   
9                    월별           08월        517         24           28.1   
10                   월별           09월        501       25.4           43.5   
11                   월별           10월        522       22.1           43.5   
12                   월별           11월        478       31.1           35.3   
13                   월별           12월        467         43           34.4   
14                  거주지         서울특별시       1424         27           30.2   
15                  거주지         부산광역시        697       25.5           38.2   
16                  거주지         대구광역시        512       32.1           28.1   
17                  거주지         인천광역시        353       31.4           29.3   
18                  거주지         광주광역시        205       22.2           31.3   
19                  거주지         대전광역시        168       29.6           32.3   
20                  거주지         울산광역시        176       23.3           38.9   
21                  거주지         세종자치시         32       14.4           26.6   
22                  거주지           경기도       1226       27.7           30.8   
23                  거주지           강원도         95       24.5           31.4   
24                  거주지           충청도        239       23.3           32.9   
25                  거주지           전라도        361       25.5           37.8   
26                  거주지           경상도        385       23.1           31.2   
27                  거주지  해외거주 등 기타 지역         11       23.3           21.2   
28                   성별            남자       2456       24.3           29.3   
29                   성별            여자       3428       28.7             34   
30                   연령     15세 - 20세        103       28.7           15.7   
31                   연령     21세 - 30세       1388       31.9           27.3   
32                   연령     31세 - 40세       1491       30.8           29.4   
33                   연령     41세 - 50세       1202       22.3             34   
34                   연령     51세 - 60세       1267       22.1           37.5   
35                   연령        60세 이상        433       23.4           38.5   
36                 방문목적     휴가 및 순수여행       5329       28.3           34.2   
37                 방문목적        비즈니스여행        359       13.3            9.8   
38                 방문목적          기타여행        196       14.2           13.9   
39                 여행형태          개별여행       5884       26.9             32   
40                 방문횟수            1회       1833       27.4           35.3   
41                 방문횟수            2회       2370       30.3           34.3   
42                 방문횟수            3회        773       24.5           29.8   
43                 방문횟수         4회 이상        908       18.8           21.5   
44  ♠BASE : 13번의 1번 응답자           NaN        NaN        NaN            NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   5만원 이상-10만원 미만  10만원 이상-20만원 미만    20만원 이상       경험없음       [평균]  
1             11.8              2.7        0.6         26   23691.87  
2             10.3              3.6        0.6       51.8   16985.87  
3               10              2.9        0.8       49.1   16992.76  
4              7.9              2.7        0.3       35.3   18269.74  
5  

In [ ]:
culture_cost_2018 = culture_cost_2018.drop(index = range(14, 45)).reset_index(drop=True)

culture_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4      Unnamed: 5  \
0    NaN        NaN        사례수     2만원 미만  2만원 이상-5만원 미만  5만원 이상-10만원 미만   
1   [전체]        NaN       5884       26.9             32            11.8   
2     월별        01월        438       20.5           13.1            10.3   
3     월별        02월        414       21.4           15.9              10   
4     월별        03월        466       28.2           25.6             7.9   
5     월별        04월        520       27.5           39.3            16.2   
6     월별        05월        510       21.1           38.8            16.2   
7     월별        06월        535       24.8           34.9            14.1   
8     월별        07월        517       33.3           25.8             6.1   
9     월별        08월        517         24           28.1            15.7   
10    월별        09월        501       25.4           43.5            11.7   
11    월별        10월        522       22.1           43.5            12.4   
12    월별        11월        478       31.1           35.3            11.6   
13    월별        12월        467         43           34.4             8.4   

         Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   10만원 이상-20만원 미만    20만원 이상       경험없음       [평균]  
1               2.7        0.6         26   23691.87  
2               3.6        0.6       51.8   16985.87  
3               2.9        0.8       49.1   16992.76  
4               2.7        0.3       35.3   18269.74  
5               1.6        0.3       15.1    25831.9  
6               3.8        1.2       18.9   33653.54  
7               3.5        1.1       21.6   27628.53  
8               3.2          1       30.6   20928.59  
9               3.5        0.5       28.2   25974.52  
10              3.5        0.2       15.6   26784.51  
11              1.4          0       20.7   23526.46  
12              1.6        0.7       19.6   23664.18  
13                1        0.3       12.8   20992.83

In [ ]:
culture_cost_2018 = culture_cost_2018.drop(index = [0, 1]).reset_index(drop=True)

culture_cost_2018

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0     월별        01월        438       20.5       13.1       10.3        3.6   
1     월별        02월        414       21.4       15.9         10        2.9   
2     월별        03월        466       28.2       25.6        7.9        2.7   
3     월별        04월        520       27.5       39.3       16.2        1.6   
4     월별        05월        510       21.1       38.8       16.2        3.8   
5     월별        06월        535       24.8       34.9       14.1        3.5   
6     월별        07월        517       33.3       25.8        6.1        3.2   
7     월별        08월        517         24       28.1       15.7        3.5   
8     월별        09월        501       25.4       43.5       11.7        3.5   
9     월별        10월        522       22.1       43.5       12.4        1.4   
10    월별        11월        478       31.1       35.3       11.6        1.6   
11    월별        12월        467         43       34.4        8.4          1   

   Unnamed: 7 Unnamed: 8 Unnamed: 9  
0         0.6       51.8   16985.87  
1         0.8       49.1   16992.76  
2         0.3       35.3   18269.74  
3         0.3       15.1    25831.9  
4         1.2       18.9   33653.54  
5         1.1       21.6   27628.53  
6           1       30.6   20928.59  
7         0.5       28.2   25974.52  
8         0.2       15.6   26784.51  
9           0       20.7   23526.46  
10        0.7       19.6   23664.18  
11        0.3       12.8   20992.83

1) 월별 패턴:

- 12월에는 2만원 미만의 범주가 높고, 5월에는 20만원 이상의 범주가 높음.

2) 평균 금액:

- 5월에 평균 금액이 가장 높고, 1월과 12월이 평균 금액이 가장 낮음.

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '2만원 미만',
           'Unnamed: 4' : '2만원 이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상',
           'Unnamed: 8' : '경험없음',
           'Unnamed: 9' : '평균'}

culture_cost_2018 = culture_cost_2018.rename(columns = columns)
culture_cost_2018

분석 카테고리 세부 카테고리  사례수 2만원 미만 2만원 이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0       월별     01월  438   20.5          13.1           10.3             3.6   
1       월별     02월  414   21.4          15.9             10             2.9   
2       월별     03월  466   28.2          25.6            7.9             2.7   
3       월별     04월  520   27.5          39.3           16.2             1.6   
4       월별     05월  510   21.1          38.8           16.2             3.8   
5       월별     06월  535   24.8          34.9           14.1             3.5   
6       월별     07월  517   33.3          25.8            6.1             3.2   
7       월별     08월  517     24          28.1           15.7             3.5   
8       월별     09월  501   25.4          43.5           11.7             3.5   
9       월별     10월  522   22.1          43.5           12.4             1.4   
10      월별     11월  478   31.1          35.3           11.6             1.6   
11      월별     12월  467     43          34.4            8.4               1   

   20만원 이상  경험없음        평균  
0      0.6  51.8  16985.87  
1      0.8  49.1  16992.76  
2      0.3  35.3  18269.74  
3      0.3  15.1   25831.9  
4      1.2  18.9  33653.54  
5      1.1  21.6  27628.53  
6        1  30.6  20928.59  
7      0.5  28.2  25974.52  
8      0.2  15.6  26784.51  
9        0  20.7  23526.46  
10     0.7  19.6  23664.18  
11     0.3  12.8  20992.83